### ISSUE TITLE
Mismatch touchID

### ISSUE THREAD

**Issue by artola** [mCodex/react-native-sensitive-info#263]
In documentation "Recipes - Fingerprint Protection #Saving" say `touchId` (see [link](https://mcodex.dev/react-native-sensitive-info/docs/protectingWithFingerprint#saving)) while in same document under "#Getting" or in examples and typings correctly shows `touchID`.

**Comment by gabrielmoncea** [comment]
@artola correct property is `touchID`

**Comment by artola** [comment]
> @artola correct property is `touchID`

Right, as stated above `typings correctly shows touchID.`, just docs must be fixed to reflect in the recipes: https://mcodex.dev/react-native-sensitive-info/docs/protectingWithFingerprint/#saving
